src/pages/LandingPages/CorporateCampaign/InContextCheckout.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+	<div>
+		<kv-lightbox
+			:visible="lightboxClosed"
+			@lightbox-closed="emit('lightbox-closed')"
+			title="Complete your loan"
+		>
+			<!-- :teams="teams" -->
+			<basket-items-list
+				:loans="loans"
+				:donations="donations"
+				:kiva-cards="kivaCards"
+				:loan-reservation-total="parseInt(totals.loanReservationTotal)"
+			/>
+			<hr>
+			<template v-slot:controls>
+				<kv-button
+					v-if="!isActivelyLoggedIn"
+					class="smaller checkout-button"
+					id="Continue-to-legacy-button"
+					v-kv-track-event="['basket', 'Redirect Continue Button', 'exit to legacy']"
+					:href="registerOrLoginHref"
+				>
+					Continue
+				</kv-button>
+
+				<kiva-credit-payment
+					v-else
+					@complete-transaction="completeTransaction"
+					class=" checkout-button"
+					id="kiva-credit-payment-button"
+				/>
+			</template>
+		</kv-lightbox>
+	</div>
+</template>
+
+<script>
+import numeral from 'numeral';
+import checkoutUtils from '@/plugins/checkout-utils-mixin';
+import KivaCreditPayment from '@/components/Checkout/KivaCreditPayment';
+import BasketItemsList from '@/components/Checkout/BasketItemsList';
+import KvButton from '@/components/Kv/KvButton';
+import KvLightbox from '@/components/Kv/KvLightbox';
+
+export default {
+	components: {
+		BasketItemsList,
+		KvButton,
+		KvLightbox,
+		KivaCreditPayment,
+	},
+	mixins: [
+		checkoutUtils
+	],
+	props: {
+		lightboxClosed: {
+			type: Boolean,
+			default: false
+		},
+		isActivelyLoggedIn: {
+			type: Boolean,
+			default: false
+		},
+		totals: {
+			type: Object,
+			default: () => {},
+		},
+		loans: {
+			type: Array,
+			default: () => [],
+		},
+		donations: {
+			type: Array,
+			default: () => [],
+		},
+		kivaCards: {
+			type: Array,
+			default: () => [],
+		}
+	},
+	data() {
+		return {
+
+		};
+	},
+	computed: {
+		registerOrLoginHref() {
+			if (typeof window !== 'undefined') {
+				return `/ui-login?force=true&doneUrl=${window.location.href}`;
+			}
+			// TODO: map using route for ssr
+			return '/ui-login?force=true';
```

Comment:
For both client and server you can use
```
return `/ui-login?force=true&doneUrl=${this.$route.path}`
```

Reply:
Cool thanks. I need to make sure all query params will flow through as well and will work that out with the router data.

Reply:
Good point, you might need to do `encodeURIComponent(this.$route.fullPath)`